addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -35,7 +35,7 @@
     <t t-name="website.form_field">
         <div t-attf-class="s_website_form_field mb-3 #{field.formatInfo.col or 'col-12'} #{field.custom and 's_website_form_custom' or ''} #{(field.required and 's_website_form_required' or '') or (field.modelRequired and 's_website_form_model_required' or '')} #{field.hidden and 's_website_form_field_hidden' or ''} #{field.dnone and 's_website_form_dnone' or ''}"
             t-att-data-type="field.type"
-            data-name="Field">
+            t-att-data-name="defaultName">
```

Comment:
You cannot do that in a stable version. You can *add* thing to the XML but not change/remove things. If a custo is xpath this element, you are breaking it.

It would be very easy to imagine someone making a custo on this for the same reason as you fixing the translation bug: they could target `//div[@data-name]` to add a `data-translated-name` for instance -> you remove it -> crash.

Sure, this is probably not the case. But if everyone does this in all their fixes, surely our thousands (millions?) of custo would crash every week :wink: 

In this case, to test, but I think that if you leave the data-name untouched and add a t-add-data-name alongside it, it works. Then in the master version, you would remove the useless one.

Reply:
Makes total sense. I tried your suggestion to have both the `data-name` and `t-att-data-name` attributes, but that doesn't seem to work. It takes the `data-name`. So I tried adding a `t-att-data-translated-name` here, and modified the code that looks for the name to put in the sidebar title. Let me know if you think it's fine like this.